disco_aws_automation/disco_deploy.py

```diff
@@ -51,9 +52,15 @@ def _get_hostclasses_from_pipeline_definition(self, pipeline_definition):
         hostclasses = {entry["hostclass"]: entry for entry in pipeline_definition}
 
         for hostclass, entry in hostclasses.iteritems():
-            for definition in ["max_size", "desired_size", "min_size"]:
-                if definition in entry:
-                    hostclasses[hostclass][definition] = int(entry[definition])
+            if "min_size" in entry:
```

Comment:
so, i originally had it so that these values and the logic to calculate it was encapsulated in a separate Pipeline class that can then be re-used throughout this script and disco_aws.py (#31), but that ended up touching too many things and I was asked to just make the fix here.